tests/pipelines/stable_diffusion_xl/test_stable_diffusion_xl.py

```diff
@@ -938,36 +938,57 @@ def test_stable_diffusion_xl_save_from_pretrained(self):
 
         assert np.abs(image_slices[0] - image_slices[1]).max() < 1e-3
 
-    def test_stable_diffusion_xl_with_fused_qkv_projections(self):
```

Comment:
Why is this test being deleted?

Reply:
Oh this is a mistake. Good catch. Fixing

Reply:
Fixed